src/alcotest-engine/config_intf.ml

```diff
@@ -13,7 +13,8 @@ module Types = struct
     ; filter : filter option
     ; log_dir : string
     ; bail : bool
-    ; record_backtrace : bool >
+    ; record_backtrace : bool
+    ; ci : [ `Github_actions | `OCamlci | `Unknown | `Disabled ] >
```

Comment:
Could you declare this as a separate type?

Reply:
Done, thanks!